eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/embed.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set content_name=ez_content_name(content) %}
+
+{% if location is defined %}
```

Comment:
Even though this still needs to be fixed...

The last plan was to *always* make the location available, as it will very often be needed, and getting it isn't that simple (custom controller). But we will also add a property that says if the location was loaded by default, or explicitly requested.